src/BackgroundEffects.js

```diff
@@ -122,57 +83,23 @@ module.exports = class Effects {
     this.clouds = clouds(p5, lerpColorFromPalette);
     this.frosted_grid = frostedGrid(p5, getCurrentPalette, randomNumber);
     this.starburst = starburst(p5, lerpColorFromPalette, randomColorFromPalette, randomNumber);
-    this.diamonds = diamonds(p5, lerpColorFromPalette, getInPreviewMode);
+    this.diamonds = diamonds(p5, lerpColorFromPalette, getEffectsInPreviewMode);
     this.circles = circles(p5, lerpColorFromPalette);
     this.sparkles = sparkles(p5, randomColorFromPalette, randomNumber);
-    this.text = text(p5, lerpColorFromPalette, colorFromHue, randomNumber, getInPreviewMode);
+    this.text = text(p5, lerpColorFromPalette, colorFromHue, randomNumber, getEffectsInPreviewMode);
     this.splatter = splatter(p5, randomColorFromPalette);
     this.swirl = swirl(p5, randomColorFromPalette);
     this.spiral = spiral(p5, lerpColorFromPalette);
-    this.lasers = lasers(p5, lerpColorFromPalette, getInPreviewMode);
+    this.lasers = lasers(p5, lerpColorFromPalette, getEffectsInPreviewMode);
```

Comment:
I think you mentioned adding the user-facing names for effects as comments for easier reference?